examples/with-typescript2/tsconfig.json

```diff
@@ -0,0 +1,26 @@
+{
+  "compileOnSave": false,
+  "compilerOptions": {
+    "target": "esnext",
+    "module": "esnext",
+    "jsx": "preserve",
```

Comment:
Do we still need to preserve JSX?

Reply:
this option from <https://github.com/zeit/next-plugins/tree/master/packages/next-typescript>

I’ve tested styled-jsx with jsx option removal,
It works well, I removed from code